exist-versioning-release.md

```diff
@@ -93,40 +93,121 @@ That version number is also copied into the `META-INF/MANIFEST.MF` file of any J
 
 ## Release Process
 
-This section details concrete steps for creating and publishing product releases. Each section here assumes you are starting with a clean Git checkout of the `develop` branch from [https://github.com/eXist-db/exist.git](https://github.com/eXist-db/exist.git).
+This section details concrete steps for creating and publishing product releases. 
 
-### Initiating Semantic Versioning
+### Release preprations and prerequisites 
 
-Version 3.0.0 was released before Semantic Versioning. The following steps will initiate Semantic Versioning for the remainder of the development phase of the next release, a new minor version to be called version 3.1.0:
+#### Required Software 
+Once development on a new stable version is complete, the following steps will prepare the version for release. For purposes of illustration, we will assume we are preparing the stable release of version 5.3.0.
+You will require a system with:
+
+* macOS
+* JDK 8
+* Maven 3.6.0+
+* Docker
+* GnuPG
+
+#### Required Keys, Credentials and Permissions
+
+##### GPG Key / Keystore and Credentials
+A private GPG key for signing release artifacts is required. To gather the required information run `gpg --list-secret-keys --keyid-format SHORT`
```

Comment:
"A private GPG key for signing release artifacts" should be changed to:

"A Public/Private PGP key pair is needed for signing release artifacts. Such a key/pair can be generated with GPG if desired. Regardless, the public key MUST have been published to the public key server network, so that it may be verified by Maven Central when the artifacts are published".